pandas/plotting/_core.py

```diff
@@ -2078,27 +2067,27 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
         :context: close-figs
 
         >>> np.random.seed(1234)
-        >>> df = pd.DataFrame(np.random.rand(10,4),
+        >>> df = pd.DataFrame(np.random.randn(10,4),
```

Comment:
flake8 will demand a space after the comma, so `np.random.randn(10, 4)`

Reply:
It accepts it, no error popped out!